web/src/components/avatar/styles/item.ts

```diff
@@ -0,0 +1,68 @@
+import { JssStyle } from 'jss'
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../../tokens'
+import { parseToken } from '../../../utils'
+
+export const useItemStyle = createUseStyles(
+  (theme: typeof tokens): Record<string, JssStyle> => ({
+    wrapper: {
+      display: 'flex',
+      border: 0,
+      padding: theme.sizeHalfX,
+      alignItems: 'center',
+      backgroundColor: 'transparent',
+      cursor: ({ isDisabled }) => (isDisabled ? 'auto' : 'pointer'),
+      '& > div:first-child': {
+        transition: 'all 0.25s ease-in-out',
+      },
+      '&:hover': {
+        '& > div:first-child': {
+          transform: ({ isInactive, isDisabled }) =>
+            isInactive || isDisabled ? 'none' : 'scale(1.1)',
+        },
+        '& > div:first-child > div:nth-child(2)': {
+          transform: ({ isInactive, isDisabled }) =>
+            isInactive || isDisabled ? 'none' : 'scale(0.91)',
+        },
+      },
```

Comment:
same here. scale the circle instead of the wrapper!

Reply:
Agree, just pushed changes. Thanks!